presto-spi/src/main/java/com/facebook/presto/spi/statistics/Estimate.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+import java.util.function.Function;
+
+import static java.lang.Double.isNaN;
+
+public final class Estimate
+{
+    // todo eventually add some notion of statistic reliability
+    //      Skipping for now as there hard to compute it properly and so far we do not have
```

Comment:
The comment is strange because this is for the SPI and "hard to compute it properly" is a connector-specific concern.